File: projects/web/java/spring/{{cookiecutter.project_slug}}/src/main/java/{{cookiecutter.java_package_dir}}/api/PingController.java
package {{ cookiecutter.java_package }}.api;

import {{ cookiecutter.java_package }}.Version;
import {{ cookiecutter.java_package }}.dto.Health;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

/** Ping. Every API exposes it at the root. */
@RestController
public class PingController {
    @GetMapping("/ping")
    public Health ping() {
        return new Health("ok", Version.VERSION);
    }
}

File: projects/web/kotlin/spring/{{cookiecutter.project_slug}}/src/main/kotlin/{{cookiecutter.java_package_dir}}/api/PingController.kt
package {{ cookiecutter.java_package }}.api

import {{ cookiecutter.java_package }}.Version
import {{ cookiecutter.java_package }}.dto.Health
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

/** Ping. Every API exposes it at the root. */
@RestController
class PingController {
    @GetMapping("/ping")
    fun ping() = Health("ok", Version.VERSION)
}
